FF1Lib/QualityOfLife.cs

```diff
@@ -45,6 +77,75 @@ public void DisableDamageTileSFX()
 		{
 			Put(0x7C7E7, Blob.FromHex("EAEAEAEAEAEAEAEAEAEAEAEAEAEAEA"));
 		}
+
+		public void SetAltPoisonSFX(Flags flags, PoisonSFX poisonSFX)
+		{
+
+			string SFXCommand;
+			switch (poisonSFX)
+			{
+				case PoisonSFX.Silent:
+				{
+					SFXCommand = "EAEAEAEAEAEAEAEAEAEAEAEAEAEAEAEAEAEAEAEAEAEAEAEA";
+					break;
+				}
+				case PoisonSFX.Ouch:
+				{
+					// one iteration of the vanilla poison sound
+					SFXCommand = "A93F8D0440A9818D0540A9608D0640A9008D0740A906857E";
+					break;
+				}
+				case PoisonSFX.Beep:
+				{
+					if (flags.TournamentSafe)
+					{
+						// Beep sound was deemed not tournament safe; might remind spectators of hospital sounds
+						// Set to "Ouch" instead.
+						SFXCommand = "A93F8D0440A9818D0540A9608D0640A9008D0740A906857E";
+					}
+					else
+					{
+						// similar to the Zelda 1 low-health alarm
+						SFXCommand = "A9BA8D0440A9008D0540A95E8D0640A9008D0740A906857E";
+					}
+					break;
+				}
+				case PoisonSFX.Bonk:
+				{
+					// similar to the Dragon Warrior "bonk into wall" sound
+					SFXCommand = "A9BF8D0440A9B48D0540A9F08D0640A9028D0740A909857E";
+					break;
+				}
+				case PoisonSFX.Oops:
+				{
+					// similar to the Super Mario Bros. 1 stomp/swim sound
+					SFXCommand = "A9BA8D0440A98C8D0540A9FF8D0640A9008D0740A90A857E";
+					break;
+				}
+				default:
+				{
+					/// not likely ever to end up here, but just in case...
+					SFXCommand = "EAEAEAEAEAEAEAEAEAEAEAEAEAEAEAEAEAEAEAEAEAEAEAEA";
+					break;
+				}
+
+			}
+
+			// Overwrites the entire MapPoisonDamage subroutine, preserving damage assignment but changing the order
+			// of operations, playing a new sound (or none) once every 4 steps instead of once every frame
+			PutInBank(0x1F, 0xC7FB, Blob.FromHex($"A534F00160A000A200BD0161C903D01FA001BD0B61D007BD0A61C9029011BD0A6138E9019D0A61BD0B61E9009D0B618A186940AAD0D398F01FADA0602903D018{SFXCommand}60EAEAEAEAEAEAEAEAEAEAEAEAEA"));
+		}
+
+		public void DisableAirshipSFX()
+		{
+			// this is very simple, so no .asm saved for this routine
+		
+			// NOP out the sfx during the airship transition animations
+			PutInBank(0x1F,0xE215,Blob.FromHex("60EAEAEAEAEAEAEAEAEAEAEAEAEAEAEA"));
```

Comment:
how does it interact with airboat?

Reply:
If you need confirmation in the code, airboat goes through the vanilla animation and movement routines once you initiate flight/landing, so skipping the sfx in those doesn't alter any other behavior

https://github.com/FiendsOfTheElements/FF1Randomizer/blob/4fb5e0ca29f0f63ed9c22fe4e67c090cf20e33ea/FF1Lib/asm/1B_A000_AirBoatRoutines.asm#L217